### PR TITLE
add one more routed network to static_route_entry sample playbook

### PR DESCRIPTION
**Pull request by 119Vik**
Added one more sample playbook to address issue #92

### CODE REVIEW

File: samples/ftd_configuration/static_route_entry.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - hosts: all
   connection: httpapi
   tasks:
-    - name: Setup destination network for static route
+    - name: Setup first destination network for static route
```
Comment (annikulin) [inline samples/ftd_configuration/static_route_entry.yml]:
```suggestion
    - name: Setup the first destination network for static route
```

File: samples/ftd_configuration/static_route_entry.yml [hunk 2]
```diff
@@ -10,6 +10,15 @@
           value: 192.168.4.0/24
           dnsResolution: IPV4_AND_IPV6
           type: networkobject
+    - name: Setup second destination network for static route
```
Comment (annikulin) [inline samples/ftd_configuration/static_route_entry.yml]:
```suggestion
    - name: Setup the second destination network for static route
```
And let's add an empty line above.

File: samples/ftd_configuration/static_route_entry.yml (hunk 1 above)
Comment (119Vik) [reply samples/ftd_configuration/static_route_entry.yml]: fixed

File: samples/ftd_configuration/static_route_entry.yml (hunk 2 above)
Comment (119Vik) [reply samples/ftd_configuration/static_route_entry.yml]: fixed

**Pull request merged**